plugins/tiddlywiki/savetrail/savetrail.js

```diff
@@ -99,7 +99,14 @@ SaveTrailSyncAdaptor.prototype.getTiddlerInfo = function(tiddler) {
 /*
 Save a tiddler and invoke the callback with (err,adaptorInfo,revision)
 */
-SaveTrailSyncAdaptor.prototype.saveTiddler = function(tiddler,callback) {
+SaveTrailSyncAdaptor.prototype.saveTiddler = function(tiddler,options,callback) {
+	if(!!callback && typeof callback !== "function"){
+		var optionsArg = callback;
+	}
+	if(typeof options === "function"){
+		callback = options;
+		options = optionsArg || {};
+	}
```

Comment:
Please add 1 or 2 lines of code-comments, that this is done for compatibility reasons. Right? Starting with 5.1.24.